<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowserWindow</class>
 <widget class="QMainWindow" name="BrowserWindow">
  <widget class="QWidget" name="centralWidget">
   <widget class="NowPlayingIndicator" name="indicator" native="true"/>
   <layout class="QVBoxLayout" name="centralLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="FastListView" name="objectList">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="searchWidget" native="true">
      <property name="visible">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="searchLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="searchEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="searchHideButton"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <widget class="KbMenu" name="windowMenu">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <addaction name="windowMenu"/>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NowPlayingIndicator</class>
   <extends>QWidget</extends>
   <header>nowplayingindicator.h</header>
  </customwidget>
  <customwidget>
   <class>FastListView</class>
   <extends>QListView</extends>
   <header>fastlistview.h</header>
  </customwidget>
  <customwidget>
   <class>KbMenu</class>
   <extends>QMenu</extends>
   <header>kbmenu.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
